cas-server-support-osf/src/main/java/io/cos/cas/authentication/OpenScienceFrameworkCredential.java

```diff
@@ -62,8 +63,11 @@
     /** The Institution Id. */
     private String institutionId;
 
-    /** The Authentication Headers. */
-    private Map<String, String> authenticationHeaders = new HashMap<>();
+    /** The Authentication Delegation Protocol. */
+    private String delegationProtocol;
```

Comment:
Add `delegationProtocol` (above) and rename `authenticationHeaders` to `delegationAttributes` (below) for `OpenScienceFrameworkCredential`, where now there are four institution related properties:
- `remotePrincipal`
- `institutionId`
- `delegationProtocol`
- `delegationAttributes`